spec/features/user_views_suggestions_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.feature "user views suggestions", :js do
+  scenario "successfully" do
+    user = FactoryGirl.create(:user)
+    activities = FactoryGirl.create_pair(:activity, user: user)
+
+    token = double(SpotifyAuthenticator)
+    allow(SpotifyAuthenticator).to receive(:new).and_return(token)
+    allow(token).to receive(:authenticate).and_return(token)
+
+    track = {
+      "image": "https://i.scdn.co/image/d0186ad64df7d6fc5f65c20c7d16f4279ffeb815",
+      "title": "Song Title",
+      "artist": "The Killers",
+      "album": "Hot Fuss"
+    }
+    tracks = double(SuggestionsCreator, token: token)
+    allow(SuggestionsCreator).to receive(:new).and_return(tracks)
+    allow(tracks).to receive(:get_tracks).and_return(track)
+
+    visit dashboard_path(as: user)
+    job_status = SuggestionsJobStatus.create!
+    GetSuggestionsJob.perform_now(job_status)
```

Comment:
Do we need to create and perform this job? It's handled by the `show` action right?